src/jobservice/api/handler.go

```diff
@@ -278,8 +278,8 @@ func (dh *DefaultHandler) handleJSONData(w http.ResponseWriter, req *http.Reques
 
 	logger.Debugf("Serve http request '%s %s': %d %s", req.Method, req.URL.String(), code, data)
 
-	w.Header().Set(http.CanonicalHeaderKey("Accept"), "application/json")
-	w.Header().Set(http.CanonicalHeaderKey("content-type"), "application/json")
+	w.Header().Set("Accept", "application/json")
+	w.Header().Set("content-type", "application/json")
```

Comment:
"Content-Type" ?

Reply:
it's case insensitive: https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive
for consistency purpose, updated it as you suggest.
Thanks